cppcache/test/CacheableStringEqualityTest.cpp

```diff
@@ -21,9 +21,10 @@
 #include <unordered_set>
 #include <unordered_map>
 
-#include <geode/utils.hpp>
 #include <geode/CacheableString.hpp>
 
+#include "util/functional.hpp"
```

Comment:
While I like the rename, I think this one header should probably remain public. It provides the hash and equal_to for dereferenced `shared_ptr`objects. It could be quite helpful. Though I suppose we can move it public later if people are asking for it.

Reply:
Unless you have strong opinion, I think I am going to stick with your second approach to keep it private until people are asking for it.